cloud/user/user.go

```diff
@@ -467,3 +484,157 @@ func GetUser(client astrocore.CoreClient, userID string) (user astrocore.User, e
 
 	return user, nil
 }
+
+func AddDeploymentUser(email, role, deployment string, out io.Writer, client astrocore.CoreClient) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+
+	// Get all org users. Setting limit to 1000 for now
+	users, err := GetOrgUsers(client)
+	if err != nil {
+		return err
+	}
+	userID, email, err := getUserID(email, users, "organization")
+	if err != nil {
+		return err
+	}
+	mutateUserInput := astrocore.MutateDeploymentUserRoleRequest{
+		Role: role,
+	}
+	resp, err := client.MutateDeploymentUserRoleWithResponse(httpContext.Background(), ctx.Organization, deployment, userID, mutateUserInput)
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "The user %s was successfully added to the deployment with the role %s\n", email, role)
+	return nil
+}
+
+func UpdateDeploymentUserRole(email, role, deployment string, out io.Writer, client astrocore.CoreClient) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+
+	// Get all org users. Setting limit to 1000 for now
+	users, err := GetDeploymentUsers(client, deployment, userPagnationLimit)
+	if err != nil {
+		return err
+	}
+	userID, email, err := getUserID(email, users, "deployment")
+	if err != nil {
+		return err
+	}
+	mutateUserInput := astrocore.MutateDeploymentUserRoleRequest{
+		Role: role,
+	}
+	fmt.Println("deployment: " + deployment)
```

Comment:
is this here on purpose?

Reply:
the update workspace user role flow has a similar line. Not sure that we really need it though.

Reply:
removed I don't think we need it